[WPE][cross-toolchain-helper] Add sysprof and userland/vcgencmd tools into the image

clopez opened the pull request:
#### 26796239d990f7b51f4dc9418a764feb900d469b
<pre>
[WPE][cross-toolchain-helper] Add sysprof and userland/vcgencmd tools into the image
<a href="https://bugs.webkit.org/show_bug.cgi?id=277075">https://bugs.webkit.org/show_bug.cgi?id=277075</a>

Reviewed by Nikolas Zimmermann.

Add a few debugging utilities into the image by default that will be helpful to debug
issues, like

 - sysprof (the last stable version as of today, which is 3.48.0)
 - rpi userland tools: include a bunch of cli tools, like vcgencmd that allow to tune
   features like voltages and internal settings of the SoC.
   This seems to work also with the open source stack.
 - weston: enable RDP plugin to make easier see the remote scren

* Tools/yocto/meta-openembedded_and_meta-webkit.patch: update sysprof recipe.
* Tools/yocto/riscv/bblayers.conf: we now need meta-gnome for sysprof.
* Tools/yocto/riscv/manifest.xml: update to last version of meta-webkit that is the
  commit adding sysprof, userland and enabling rdp for weston.
  See: <a href="https://github.com/Igalia/meta-webkit/commit/d5d6eb779">https://github.com/Igalia/meta-webkit/commit/d5d6eb779</a>
* Tools/yocto/rpi/bblayers.conf: we now need meta-gnome for sysprof.
* Tools/yocto/rpi/manifest.xml: Same than the other manifest.xml file.

Canonical link: <a href="https://commits.webkit.org/281346@main">https://commits.webkit.org/281346@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/1be84ced841e1cb9c019c3fc1e17619e6ea9be4f

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/59613 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/38958 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/12137 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/63528 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/59/builds/10136 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/61742 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/46611 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/10290 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/48365 "Passed tests") | [✅ 🧪 wincairo-tests](https://ews-build.webkit.org/#/builders/60/builds/7097 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/61643 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/36361 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/51603 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/29203 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/33064 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/8854 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/9060 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/55012 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/9132 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/65260 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/87/builds/3541 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/62/builds/9021 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/55708 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/86/builds/3552 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/51595 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/55836 "Passed tests") | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/88/builds/2938 "Passed tests") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/8902 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/34772 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/35855 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/36941 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/35600 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nikolaszimmermann commented on Tools/yocto/meta-openembedded_and_meta-webkit.patch:
```diff
@@ -25,20 +25,6 @@ index f4911f3..b13abad 100644
-     libxtst \
-     libx11-locale \
-     xorg-minimal-fonts \
--    gdk-pixbuf-loader-ico \
```
This was necessary on my system to build for rpi3 - last time I tried this, four weeks ago. I think we should keep it for now.

nikolaszimmermann replied:
Self-reply: just saw that you updated meta-webkit - I assume my patch was already merged upstream.

clopez replied:
yes, it was merged -> https://github.com/Igalia/meta-webkit/commit/d5d6eb779

nikolaszimmermann reviewed (commented):
This seems incomplete, polkit is not enabled in the DISTRO_FEATURES and the vcgencmd stuff is not present in the PR.

clopez committed:
[WPE][cross-toolchain-helper] Add sysprof and userland/vcgencmd tools into the image
https://bugs.webkit.org/show_bug.cgi?id=277075

Reviewed by Nikolas Zimmermann.

Add a few debugging utilities into the image by default that will be helpful to debug
issues, like

 - sysprof (the last stable version as of today, which is 3.48.0)
 - rpi userland tools: include a bunch of cli tools, like vcgencmd that allow to tune
   features like voltages and internal settings of the SoC.
   This seems to work also with the open source stack.
 - weston: enable RDP plugin to make easier see the remote scren

* Tools/yocto/meta-openembedded_and_meta-webkit.patch: update sysprof recipe.
* Tools/yocto/riscv/bblayers.conf: we now need meta-gnome for sysprof.
* Tools/yocto/riscv/manifest.xml: update to last version of meta-webkit that is the
  commit adding sysprof, userland and enabling rdp for weston.
  See: https://github.com/Igalia/meta-webkit/commit/d5d6eb779
* Tools/yocto/rpi/bblayers.conf: we now need meta-gnome for sysprof.
* Tools/yocto/rpi/manifest.xml: Same than the other manifest.xml file.

Canonical link: https://commits.webkit.org/281346@main
